OreNoTask/app/controllers/admin/users_controller.rb

```diff
@@ -47,6 +47,13 @@ def create
     end
 
     def destroy # rubocop:disable Metrics/AbcSize
+      @user = User.active.find_by(id: params[:id])
+
+      if @user.privilege == 1 && User.admin_user_count == 1
```

Comment:
Enumの仕様をうまく使うと以下のように書き換えができそうですー
参考）https://api.rubyonrails.org/v5.2.5/classes/ActiveRecord/Enum.html

```suggestion
      if @user.privilege? && User.admin_user_count == 1
```

Reply:
同コントローラーの`update`のアクションでも確認しなければ行けない内容ではあるので`User`クラスのインスタンスメソッドとして定義してもいいと思います！

Reply:
enumrizeと宣言が被ってしまうので（被っても正しく動くけどrubocopに怒られる）、privilege == 1はそのままにしてメソッド化を行いました。
https://github.com/Fablic/training/pull/1187/commits/876fcf08fee9dbceae9516bfa339ea13fe57d512